src/components/Home/Home.jsx

```diff
@@ -46,6 +48,8 @@ const Home = () => {
       >
         <Icon icon={PlusIcon} />
       </Fab>
+
+      {isDialogModalOpen && <Stepper />}
```

Comment:
We've to be careful here in the future. Since we display the modal over the previous content, this previous content can be updated by the actions done by the user in the Modal. `request` results will change triggering a re-render of the `PapersList`. If the work done by `PapersList` is heavy, we'll notice a few slowdowns even in the Modal. 

So I'm wondering if we should not mount the Stepper in a new screen / route

Reply:
Je pense comprendre ton alerte, je peux en effet conditionner le contenu en fonction de `isDialogModalOpen` (ou créer une route spécifique).
Mais je ne pense pas que ça soit nécessaire, nous sommes censés enregistrer les données qu'à la fin du process, donc aucunes requêtes ne devraient partir en cours, et donc rafraîchir la `PaperList`

Reply:
Je fais suivre une PR pour corriger ça